.github/labeler.yml

```diff
@@ -49,8 +49,8 @@
   - packages/webview_flutter/**/*
 
 # Tools
-'tools':
-  - tools/**/*
+'tool':
+  - tool/**/*
```

Comment:
`tools` looks better for me :(

Reply:
I've changed it because the directory holds a single package but it's fine either way, I'll change it back.